README.md

```diff
@@ -7,6 +7,10 @@ An api with Greetings for welcome page.
  [API Docs](https://hello-rails-backendapi.herokuapp.com/api-docs/index.html)
 
  [Greetings JSON](https://hello-rails-backendapi.herokuapp.com/api/v1/greetings)
+
+ ## React App
+
+ [hello-rails-front-end](https://github.com/KDlamini/hello-rails-front-end)
 ## Built With
 
```

Comment:
- [ ] Please mention in the README file on which port we need to run the backend application and also mention the right command. Suppose, to run the application on the 3001 port you have to use this command - `rails s -p 3001`.

Reply:
Hi @Dipeshtwis,

I deployed the API therefore it is receiving the JSON remotely. There's no need to run the local API, you simply need to run the React app. the `Loading... Please wait` text only shows while the API is still loading, you probably would have seen it if you had waited a little bit longer.

Do you prefer that I removed fetching from the remote URL and apply a local one instead?

Reply:
> Hi @Dipeshtwis,
> 
> I deployed the API therefore it is receiving the JSON remotely. There's no need to run the local API, you simply need to run the React app. the `Loading... Please wait` text only shows while the API is still loading, you probably would have seen it if you had waited a little bit longer.
> 
> Do you prefer that I removed fetching from the remote URL and apply a local one instead?

Hi @KDlamini,

I have waited for 5-10min but I didn't see any message there ... It was using the URL - `https://http://........com` which I don't think it was the right URL. Please check and resolve this.

Reply:
Hi @Dipeshtwis 

See the thing that is confusing me is that everything is working perfectly fine on my side. Therefore, I'm quite confused about what it is that I need to fix. I resolved the CORS issue before I requested a review.
I even documented the API:
![image](https://user-images.githubusercontent.com/38088818/163563322-0a9ffea9-f745-44a0-bec5-8281f22f1d2e.png)

It works perfectly fine with me even if you click on the link to view it on the browser. 
On the react app I have already set up the proxy using the correct URL and the correct endpoint for the API call, which works fine on my side.

Do you mind elaborating on what you mean by "It was using the URL - `https://http://........com`" where did you see that?

Reply:
Hi @KDlamini,

I just checked again and I got the message in UI after waiting some more time. I think you are good to go.

Sorry for all the inconvenience from us. Please submit another review and you are good to go. :slightly_smiling_face: